lib/OktaAuth.ts

```diff
@@ -500,7 +500,7 @@ class OktaAuth implements SDKInterface, SigninAPI, SignoutAPI {
   async isAuthenticated(): Promise<boolean> {
 
     let { accessToken, idToken } = this.tokenManager.getTokensSync();
-    const { autoRenew, autoRemove } = this.options.tokenManager || {};
+    const { autoRenew, autoRemove } = this.tokenManager.getOptions();
```

Comment:
Use getOptions to get options merged with default ones
https://github.com/okta/okta-auth-js/blob/56206fef59751c8977b678f5a6e7e48f59ae7e4a/lib/TokenManager.ts#L40-L42